snapcraft/internal/pluginhandler/_mangling.py

```diff
@@ -0,0 +1,83 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
We already have a `snapcraft.internal.mangling` module. Should this go in there?